Packs/MongoDBAtlas/ModelingRules/MongoDBAtlas/MongoDBAtlas.xif

```diff
@@ -0,0 +1,41 @@
+[MODEL: dataset="mongodb_atlas_raw"]
+filter source_log_type = "alerts"
+| alter
+	xdm.event.type = source_log_type,
+	xdm.alert.description = acknowledgementComment,
+	xdm.source.user.username = acknowledgingUsername,
+	xdm.alert.original_alert_id = coalesce(alertConfigId, id),
+	xdm.event.original_event_type = eventTypeName,
+	xdm.source.user.groups = arraycreate(groupId),
+	xdm.observer.unique_identifier = arraystring(arraydistinct(arraymap(links -> [], "@element" -> href)), "|");
+
+filter source_log_type = "events"
+| alter
+	get_linkToDetails = arraycreate(linkToDetails),
+	get_links_href = arraydistinct(arraymap(links -> [], "@element" -> href)),
+	get_username = arraystring(arraycreate(acknowledgingUsername, username), "|")
```

Comment:
Same issue with the username here

Reply:
Removed the mapping of acknowledgingUsername for alerts.